exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/join/TestMergeJoinAdvanced.java

```diff
@@ -253,4 +259,52 @@ public void testDrill4196() throws Exception {
       .baselineValues(6000*800L)
       .go();
   }
+
+  private void buildFile(String fileName, String[] data, File testDir) throws IOException {
+    try(PrintWriter out = new PrintWriter(new FileWriter(new File(testDir, fileName)))) {
+      for (String line : data) {
+        out.println(line);
+      }
+    }
+  }
```

Comment:
The above duplicates code in the hash join test. In general, duplicate code is a "bad thing." Suggestion: move this code into a utilities class; maybe one that already exists.